lib/elixir_analyzer/exercise_test/assert_call/compiler.ex

```diff
@@ -139,13 +139,29 @@ defmodule ElixirAnalyzer.ExerciseTest.AssertCall.Compiler do
     true
   end
 
+  # For erlang libraries: :math._
+  def matching_function_call?(
+        {{:., _, [module_path, _name]}, _, _args},
+        {module_path, :_}
+      ) do
+    true
+  end
+
   def matching_function_call?(
         {{:., _, [{:__aliases__, _, module_path}, name]}, _, _args},
         {module_path, name}
       ) do
     true
   end
 
+  # For erlang libraries: :math.pow
+  def matching_function_call?(
+        {{:., _, [module_path, name]}, _, _args},
+        {module_path, name}
+      ) do
+    true
+  end
+
```

Comment:
Could you add two new test cases to `test/elixir_analyzer/exercise_test/assert_call_test.exs` to test this addition?

Reply:
Excellent idea.